src/main/resources/content/k8s/do-not-allow-sharing-host-IPC-namespace.json

```diff
@@ -1 +1,8 @@
-{"severity":"medium","rule":"$.spec.template.spec.hostIPC is true","id":"344fb01c-7195-3e9f-47e1-c640733af43f","enabled":true,"resourceType":"k8s","policy":"do not allow sharing host IPC namespace"}
\ No newline at end of file
+{
+  "severity": "medium",
+  "rule": "$.spec.template.spec.hostIPC is true",
+  "id": "344fb01c-7195-3e9f-47e1-c640733af43f",
+  "enabled": true,
+  "resourceType": "k8s",
+  "policy": "do not allow sharing host IPC namespace"
```

Comment:
description field missing here as well.
Also, the policy name is not sentence case.